### PR TITLE
follow up with more scripts and docs

### PR DESCRIPTION
**Pull request by alexmatson-da**
Follows up on #323

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,22 +87,141 @@ test-daml:
...
+		$(UI_DIR)/package.json \
+		docs/local_development.md \
+		docs/damlhub_deployment.md
+	@echo "Tagged files... check results before committing"
+
 .PHONY: release
-release: test build
-	ddit release
+release: test package
+	ddit release --dry-run
```
Comment (alexmatson-da) [inline Makefile:110]: `--dry-run` will prevent this from accidentally deploying for real, until we're ready to turn it on

File: Makefile [hunk 2]
```diff
@@ -85,22 +85,124 @@ test-daml:
...
+# Clearing Trigger
+.PHONY: start-clearing-trigger
+start-clearing-trigger: |$(STATE_DIR) $(trigger_dar_src)
+	@./scripts/run-trigger.sh \
+		$$party \
+		$(trigger_dar_src) \
+		ClearingTrigger:handleClearing \
+		clearing_trigger_$$party \
+		$(STATE_DIR)
+	@echo "Starting clearing trigger for $$party..."
```
[the remark this answers is not in the document]
Comment (alexgraham-da) [reply Makefile:181]: nice!

File: docs/damlhub_deployment.md [hunk 3]
```diff
@@ -4,27 +4,23 @@
...
 
-1. Daml Connect (1.12)
-2. `make` (3.x)
-3. `yq` (3.x)
-4. `python` (3.8)
-5. `poetry` (1.0.x)
-6. `node` (14.x)
-7. `yarn` (1.22.x)
+Windows development is not fully supported yet. It may be possible to follow these steps through
+[Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/), though this is largely untested.
```
Comment (alexgraham-da) [inline docs/damlhub_deployment.md:13]: 👍 👍

File: docs/local_development.md [hunk 4]
```diff
@@ -45,49 +43,140 @@ yarn install --force --frozen-lockfile
...
+
+The Exberry integration can be started up locally. There are two components: the adapter, included
+in this repository, and the integration itself which is in its own repository.
+
+1. Clone [`daml-dit-integration-exberry`](https://github.com/digital-asset/daml-dit-integration-exberry) in a sibling directory next to `da-marketplace/`
+2. Open two terminals, one in the `da-marketplace/` directory, and one in the `daml-dit-integration-exberry/` directory.
+3. In `daml-dit-integration-exberry/`, create a file in the repo root named `int_args.yaml`. Fill it with the template below, replaced with your Exberry account details.
+
+```yaml
+"metadata":
```
Comment (alexgraham-da) [inline docs/local_development.md:148]: `clientId` and `tokenUrl` are no longer necessary

File: docs/local_development.md [hunk 5]
```diff
@@ -45,49 +43,140 @@ yarn install --force --frozen-lockfile
...
+  "clientId": "EXBERRY_CLIENTID"
+  "tradingApiUrl": "EXBERRY_TRADING_API_URL"
+  "adminApiUrl": "EXBERRY_ADMIN_API_URL"
+  "tokenUrl": "EXBERRY_TOKEN_URL"
+  "apiKey": "EXBERRY_API_KEY"
+  "secret": "EXBERRY_SECRET"
+```
+
+4. In `daml-dit-integration-exberry/`, run `make all`
+5. In `da-marketplace/`, run `daml start --sandbox-option ../daml-dit-integration-exberry/dabl-integration-exberry-0.7.3.dar`
```
Comment (alexgraham-da) [inline docs/local_development.md:161]: I wonder if we should do `./dabl-integration-exberry-{VERSION}.dar` same for the line below so we don't have to update this README every time the integration has a new version.

File: scripts/tag-versions.sh [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# This script writes changes to files to bump versions everywhere that's needed
```
Comment (alexgraham-da) [inline scripts/tag-versions.sh:1]: Nice!

**Review by alexgraham-da — APPROVED** [review]
a few comments, but looks good!

File: docs/damlhub_deployment.md [hunk 7]
```diff
@@ -36,29 +32,27 @@ Next, enter a name and a ledger name for this project:
...
+Upload the packaged `da-marketplace-0.1.8.dit` file.
 
 Click "Launch" for both the UI and the Model:
 
 ![4_launch_dar](https://user-images.githubusercontent.com/71082197/98964004-4c802580-24d6-11eb-83bf-221a9d2ce9fa.png)
 
 ### Add the parties
 
-Next, add the parties. Click on the "Live Data" tab, and first add the `Public` and `UserAdmin` parties by clicking the "Plus" next to their names:
+Next, add the parties. Click on the "Identities" tab, and add the `UserAdmin` party by clicking the "Plus" next to its name:
```
Comment (Sarahb-da) [inline docs/damlhub_deployment.md:51]: Thanks!

File: docs/local_development.md (hunk 4 above)
Comment (alexmatson-da) [reply docs/local_development.md:148]: thanks!

File: docs/local_development.md (hunk 5 above)
Comment (alexmatson-da) [reply docs/local_development.md:161]: good idea

**Review by Sarahb-da — APPROVED** [review]
oops, meant to stamp this earlier..

**Pull request merged**